doc/guide/wsgi.py.txt

```diff
@@ -6,11 +7,12 @@ from ipalib.constants import DEFAULT_CONFIG
 # by reading in the configuration file(s). The server always reads
 # default.conf and will also read in `context'.conf.
 env = Env()
-env._bootstrap(context='server', log=None)
+env._bootstrap(context='server', log=None, confdir=paths.ETC_IPA)
 env._finalize_core(**dict(DEFAULT_CONFIG))
 
 # Initialize the API with the proper debug level
-api.bootstrap(context='server', debug=env.debug, log=None) (ref:wsgi-app-bootstrap)
+api.bootstrap(context='server', confdir=paths.ETC_IPA,
+              debug=env.debug, log=None) (ref:wsgi-app-bootstrap)
```

Comment:
This file should be kept in sync with `install/share/wsgi.py`. This change does just that, there is no other intention behind it.